src/core/server/http/__tests__/http_redirect_server.test.ts

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
So uh, should this file be called https_redirect_server.test.ts instead?

Reply:
haha, yeah, silly me

Reply:
My LGTM is pending that change. ;)